app/src/pages/inside/testCaseLibraryPage/createTestCaseModal/useTestCase.ts

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useIntl } from 'react-intl';
+import { isString } from 'es-toolkit';
+
+import { projectKeySelector } from 'controllers/project';
+import { fetch } from 'common/utils';
+import { useDebouncedSpinner } from 'common/hooks';
+import { URLS } from 'common/urls';
+import { hideModalAction } from 'controllers/modal';
+import { showErrorNotification, showSuccessNotification } from 'controllers/notification';
+import { getTestCasesAction, getTestCaseByFolderIdAction } from 'controllers/testCase';
+import { createFoldersSuccessAction } from 'controllers/testCase/actionCreators';
+
+import { CreateTestCaseFormData } from '../types';
+import { createFolder, buildManualScenario, handleTestCaseError } from './testCaseUtils';
+
+export const useTestCase = (testCaseId?: number) => {
+  const { isLoading, showSpinner, hideSpinner } = useDebouncedSpinner();
+  const dispatch = useDispatch();
+  const projectKey = useSelector(projectKeySelector);
+  const { formatMessage } = useIntl();
+
+  const handleFolder = useCallback(
+    async (payload: CreateTestCaseFormData, currentFolderId?: number): Promise<number> => {
+      if (isString(payload.folder)) {
+        const folder = await createFolder(projectKey, payload.folder);
+        dispatch(createFoldersSuccessAction({ ...folder, countOfTestCases: 0 }));
+        return folder.id;
+      }
+      return payload.folder?.id || currentFolderId;
```

Comment:
```suggestion
      if (isString(payload.folder)) {
        const folder = await createFolder(projectKey, payload.folder);

        dispatch(createFoldersSuccessAction({ ...folder, countOfTestCases: 0 }));

        return folder.id;
      }

      return payload.folder?.id || currentFolderId;
```

Please add spaces after/before const, return and if blocks

Reply:
done